build.gradle.kts

```diff
@@ -237,7 +237,12 @@ subprojects {
                     inputs.property("moduleName", moduleName)
 
                     manifest {
-                        attributes("Automatic-Module-Name" to moduleName)
+                        attributes(
+                                "Automatic-Module-Name" to moduleName,
+                                "Built-By" to System.getProperty("user.name"),
+                                "Built-JDK" to System.getProperty("java.version"),
+                                "Implementation-Title" to project.name,
```

Comment:
Just realized: I think project.name will not be very helpful as it will only be the last part, like `all` or `common`. Something based on `archivesBaseName` would be better IMHO.

Reply:
Hah, same! :) Tried that, however comparing title from old artifacts (pre-kotlin) with what yields name vs archivesBaseName I realised that it was "name" previously, so used it for backwards compatibility. Coupled with "Automatic-Module-Name" it gives full data.